Event/ScopeResolveEvent.php

```diff
@@ -45,9 +45,6 @@ public function getScopes(): array
         return $this->scopes;
     }
 
-    /**
-     * @param Scope[] $scopes
-     */
     public function setScopes(Scope ...$scopes): self
     {
         $this->scopes = (array) $scopes;
```

Comment:
This type casting is not necessary, as the argument is of needed type.

Reply:
@Derkien Good catch.